demo/android/Activity/porcupine-activity-demo-app/src/main/res/drawable/ic_launcher_foreground.xml

```diff
@@ -1,19 +1,49 @@
+          android:fillColor="#FFFFFF"/>
+      <path android:pathData="M387.20312,2Q380.28125,2,374.01562,-0.453125Q367.76562,-2.921875,363,-7.625Q358.25,-12.328125,355.4375,-19.203125Q352.64062,-26.09375,352.64062,-34.765625Q352.64062,-43.59375,355.4375,-50.46875Q358.25,-57.359375,363,-62.140625Q367.76562,-66.921875,374.01562,-69.375Q380.28125,-71.84375,387.20312,-71.84375Q394.10938,-71.84375,400.375,-69.375Q406.64062,-66.921875,411.39062,-62.140625Q416.14062,-57.359375,418.95312,-50.46875Q421.76562,-43.59375,421.76562,-34.765625Q421.76562,-26.09375,418.95312,-19.203125Q416.14062,-12.328125,411.39062,-7.625Q406.64062,-2.921875,400.375,-0.453125Q394.10938,2,387.20312,2ZM387.20312,-7.828125Q392.23438,-7.828125,396.34375,-9.78125Q400.45312,-11.75,403.40625,-15.28125Q406.35938,-18.828125,407.9375,-23.75Q409.51562,-28.6875,409.51562,-34.765625Q409.51562,-40.859375,407.9375,-45.859375Q406.35938,-50.859375,403.40625,-54.484375Q400.45312,-58.109375,396.34375,-60.046875Q392.23438,-62,387.20312,-62Q382.15625,-62,378.04688,-60.046875Q373.95312,-58.109375,371,-54.484375Q368.04688,-50.859375,366.45312,-45.859375Q364.875,-40.859375,364.875,-34.765625Q364.875,-28.6875,366.45312,-23.75Q368.04688,-18.828125,371,-15.28125Q373.95312,-11.75,378.04688,-9.78125Q382.15625,-7.828125,387.20312,-7.828125Z"
+          android:fillColor="#FFFFFF"/>
+      <path android:pathData="M475.5,0.015625L475.5,-60.3125L442.95312,-60.3125L442.95312,-70L487.3125,-70L487.3125,0.015625L475.5,0.015625ZM480.10938,-84.984375Q476.07812,-84.984375,473.34375,-87.515625Q470.60938,-90.046875,470.60938,-94.234375Q470.60938,-98.4375,473.34375,-100.96875Q476.07812,-103.5,480.10938,-103.5Q484.14062,-103.5,486.875,-100.96875Q489.60938,-98.4375,489.60938,-94.234375Q489.60938,-90.046875,486.875,-87.515625Q484.14062,-84.984375,480.10938,-84.984375Z"
+          android:fillColor="#FFFFFF"/>
+      <path android:pathData="M555.03125,1.96875Q548.1094,1.96875,542.5625,-0.5Q537.03125,-2.96875,532.9844,-7.671875Q528.9531,-12.390625,526.7969,-19.203125Q524.6406,-26.03125,524.6406,-34.859375Q524.6406,-43.421875,527.2969,-50.3125Q529.96875,-57.203125,534.3594,-61.984375Q538.75,-66.765625,544.5,-69.375Q550.2656,-72,556.46875,-72Q562.7969,-72,567.7656,-69.609375Q572.7344,-67.21875,577.34375,-62.890625L577.78125,-62.890625L577.2031,-75.90625L577.2031,-103L589.0156,-103L589.0156,-0.0625L579.21875,-0.0625L578.2031,-9.015625L577.78125,-9.015625Q573.4531,-4.53125,567.46875,-1.28125Q561.5,1.96875,555.03125,1.96875ZM557.46875,-8Q567.84375,-8,577.2031,-18.59375L577.2031,-53.734375Q572.4531,-58.25,567.90625,-60.125Q563.375,-62.015625,558.7656,-62.015625Q554.15625,-62.015625,550.1875,-60.046875Q546.2344,-58.09375,543.28125,-54.53125Q540.34375,-50.984375,538.6094,-46.046875Q536.875,-41.109375,536.875,-35.015625Q536.875,-22.234375,542.28125,-15.109375Q547.6875,-8,557.46875,-8Z"
```

Comment:
Is there any way for this layout code to have fewer of these huge pathData arrays? Not really a usability issue, but this layout file isn't really human-readable anymore